Sources/TuistLoader/ProjectDescriptionHelpers/ProjectDescriptionHelpersBuilder.swift

```diff
@@ -173,6 +173,12 @@ public final class ProjectDescriptionHelpersBuilder: ProjectDescriptionHelpersBu
                 let dylibName = "lib\(name).dylib"
                 let modulePath = moduleCacheDirectory.appending(component: dylibName)
                 let module = ProjectDescriptionHelpersModule(name: name, path: modulePath)
+
+                // If the same helpers directory has been previously compiled, we just return the module, no need to recompile it.
+                if try await fileSystem.exists(modulePath) {
```

Comment:
@fortmarek Please tell me, is this correct behavior? If I'm not mistaken, the rebooting of ProjectDescriptionHelpers occurs only if the cache has been disabled. That is, if we find ourselves in this function, then we 100% always need to rebuild the helper module and not check its presence in the cache.

Reply:
I don't think that's true. I do think we should use `CachedManifestLoader` in the `Dump` and `Generate` services but it's still possible that a manifest's hash has changed _but_ the project description helpers have not. In those cases, we can still skip building the project description helpers.